src/commands/support/discontinuity.ts

```diff
@@ -5,7 +5,7 @@ import { makeEmbed, makeLines } from '../../lib/embed';
 export const discontinuity: CommandDefinition = {
     name: ['discontinuity', 'disco'],
     description: 'Link to docs about discontinuities',
-    category: CommandCategory.A32NX,
+    category: CommandCategory.SUPPORT,
```

Comment:
Any reason why this one moved to support rather than aircraft?

Reply:
It's in the support folder... So I fixed this one. We could move the entire command to the aircraft folder, though.

Reply:
Ah right yes, I think that was probably an accident when command was created? I feel it should be in aircraft - what about you?